ibis/backends/flink/tests/test_udf.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import json
+
+from ibis import udf
+
+
+def test_builtin_scalar_udf(con):
+    @udf.scalar.builtin
+    def parse_url(string1: str, string2: str) -> str:
+        ...
+
+    expr = parse_url("http://facebook.com/path1/p.php?k1=v1&k2=v2#Ref1", "HOST")
+    result = con.execute(expr)
+    assert result == "facebook.com"
+
+
+def test_builtin_agg_udf(con):
+    @udf.agg.builtin
+    def json_arrayagg(a) -> str:
+        """Glom together some JSON."""
+
+    ft = con.tables.functional_alltypes[:5]
+    expr = json_arrayagg(ft.string_col)
+    result = expr.execute()
+    expected = json.dumps(list(map(str, range(5))), separators=",:")
```

Comment:
```suggestion
    expected = '["1","2","3","4","5"]'
```

Just to make it a bit more obvious what the JSON looks like :)

Reply:
Sounds good; I'll update it here and wherever I copied it from (`mysql`, I think).